cmd/harmony/main.go

```diff
@@ -636,5 +638,25 @@ func main() {
 		go currentNode.CollectMetrics()
 	}
 
+	// Prepare for graceful shutdown from os signals
+	osSignal := make(chan os.Signal)
+	signal.Notify(osSignal, os.Interrupt, os.Kill, syscall.SIGTERM)
```

Comment:
In addition to signals, os.Interrupt, os.Kill, syscall.SIGTERM, there are:
`syscall.SIGHUP, syscall.SIGQUIT, syscall.SIGTERM, syscall.SIGINT` (from this example of handling graceful exit https://github.com/apache/dubbo-go/issues/86)
Do we also want to include these?